src/sentry/testutils/helpers/notifications.py

```diff
@@ -23,13 +24,15 @@ def build_attachment_title(self, *args):
     def get_title_link(self, *args):
         return None
 
-    def get_notification_title(self, context: Mapping[str, Any] | None = None) -> str:
+    def get_notification_title(
+        self, provider: ExternalProviders, context: Mapping[str, Any] | None = None
```

Comment:
@97varun we can leave provider as the first arg here since `context` is optional